recipes/ConservedWaterSearch/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ConservedWaterSearch" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ConservedWaterSearch-{{ version }}.tar.gz
+  sha256: 0811404f0bd2dbac693bbc2f0d6ee1163dbeee34c552c5e33e82cd00a1a56a74
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - pymol-open-source
+    - cython
```

Comment:
Are both of these really required?

Reply:
Cython is a hdbscan requirement. However, if cython is omitted from our list of requirements hdbscan building fails for some reason and the whole build fails. 

pymol-open-source is required for generation of visualization of results produced by the code.

Reply:
In that case, it should probably be added to hdbscan and not here. Please open a PR at their [feedstock](https://github.com/conda-forge/hdbscan-feedstock), seems like there is already an issue for it with a similar problem and ping core, in case the maintainer doesn't answer. 

Ref: https://conda-forge.org/docs/maintainer/maintainer_faq.html#mfaq-contact-core

Ahh, I didn't find it in the requirements upstream. If it's not there, not a problem for now but please add it there as well, as we prefer to have upstream and here in sync.

Reply:
The cython issue has been resolved on hdbscan side conda-forge/hdbscan-feedstock#54 .

We have also added pymol as a dependency upstream as well.